conf/machine/arrakis-pico.conf

```diff
@@ -0,0 +1,66 @@
+require conf/machine/genericx86-64.conf
+
+# additional kernel config
+OMNECT_KERNEL_SRC_URI:append = " \
+    file://crc32c-intel_module.cfg \
+    file://disable_wlan.cfg \
+    file://intel-lpc_ich_module.cfg \
+    file://intel-mei_module.cfg \
+    file://intel-vbtn_module.cfg \
+    file://pc-speaker.cfg \
+    file://tcg_tpm_2.0.cfg \
```

Comment:
how did you find out about the necessity of those? by defconfig comparison from yocto-linux and welotec-linux?